napari/_vispy/layers/base.py

```diff
@@ -155,10 +155,11 @@ def _on_matrix_change(self):
 
     def _on_experimental_clipping_planes_change(self):
         if hasattr(self.node, 'clipping_planes'):
-            self.node.clipping_planes = (
-                # invert axes because vispy uses xyz but napari zyx
-                self.layer.experimental_clipping_planes.as_array()[..., ::-1]
-            )
+            prev = self.node.clipping_planes
+            # invert axes because vispy uses xyz but napari zyx
+            new = self.layer.experimental_clipping_planes.as_array()[..., ::-1]
+            if prev != new:
```

Comment:
This condition compares two `np.ndarray` objects right? And they might be different shapes (at least have a different length in the first dimension). 

In that case, I think we should probably use `np.array_equal` to avoid the variety of warnings associated with array comparison.

Though, I didn't see any warnings when using a points layer. So maybe I'm missing something. Or are those being ignored/suppressed somewhere?

Reply:
You're right... Not sure why no warnings appeared actually.